Mobile/lib/services/app_state.dart

```diff
@@ -182,16 +203,16 @@ class RootAppState extends ChangeNotifier {
 
       for (var task in jsonData) {
         newTasks.add(new Task(
-          task['id'],
-          task['title'],
-          task['description'],
-          task['reward'],
-          DateTime.parse(task['start_date']),
-          DateTime.parse(task['end_date']),
-          _getBool(task['recurring']),
-          task['recurring_interval'],
-          _getBool(task['single_completion'])
-        ));
+            // task['id'],
+            1, // i don't know what happen but i don't get the id with the task 😭
```

Comment:
the tasks returned, should contain an id, instead of setting it to a static value.